addons/dragon/CfgAmmo.hpp

```diff
@@ -0,0 +1,150 @@
+        thrustTime = 0;
+        sideAirFriction = 0.05;
+        effectsMissile = "missile2";
+        effectFlare = "";
+        airFriction = 0.5;
+        fuseDistance = 75;
+        whistleDist = 2;
+        
+        indirectHit = 9;
+        indirectHitRange = 1;
+        explosive = 0.02;
+        timeToLive = 60;
+        cost = 4000;
+        simulationStep = 0.005;
+        maxControlRange = 1500;
+        
+        class ace_missileguidance {
+            minDeflection = 0;
+            maxDeflection = 0;
+            incDeflection = 0;
+
+            canVanillaLock = 0;
+
+            // Guidance type for munitions
+            defaultSeekerType = "SACLOS";
+            seekerTypes[] = { "SACLOS" };
+
+            defaultSeekerLockMode = "LOAL";
+            seekerLockModes[] = { "LOAL", "LOBL" };
+
+            seekLastTargetPos = 0;
+            seekerAngle = 30;
+            seekerAccuracy = 1;
+
+            seekerMinRange = 65;
+            seekerMaxRange = 1000;
+
+            correctionDistance = 30;
+            missileLeadDistance = 0;
+            offsetFromCrosshair[] = { 0, 0, 0 };
+            
+            serviceInterval = 0.33; // how many seconds between pops
+            serviceCharges = 32; // how many charges are in this missile
+            serviceChargeAcceleration = 6.5;
+            dragonSpeed = 100; // meters per second
+
+            defaultAttackProfile = "DRAGON";
+            attackProfiles[] = {"DRAGON"};
+        };
+    };
+    
+    class GVAR(super) : GVAR(dragonBase) {
+        scope = 1;
+        aiAmmoUsageFlags = "128+512";
+        hit = 460;
+        indirectHit = 9;
+        indirectHitRange = 1;
+        explosive = 0.02;
+        cost = 4000;
+        
+        submunitionAmmo = QGVAR(penetrator_super);
+        submunitionDirectionType = "SubmunitionModelDirection";
+        submunitionInitSpeed = 200;
+        submunitionParentSpeedCoef = 0;
+        submunitionInitialOffset[] = { 0, 0, -0.2 };
+
+        class ace_missileguidance {
+            enabled = 1;
+
+            // Guidance type for munitions
+            defaultSeekerType = "SACLOS";
+            seekerTypes[] = { "SACLOS" };
+
+            defaultSeekerLockMode = "LOAL";
+            seekerLockModes[] = { "LOAL", "LOBL" };
+
+            seekLastTargetPos = 0;
+            seekerAngle = 30;
+            seekerAccuracy = 1;
+
+            seekerMinRange = 30;
+            seekerMaxRange = 1500;
+
+            correctionDistance = 30;
+            missileLeadDistance = 0;
+            
+            serviceInterval = 0.33; // how many seconds between pops
+            serviceCharges = 65; // how many charges are in this missile
+            serviceChargeAcceleration = 10;
+            dragonSpeed = 200; // meters per second
+
+            defaultAttackProfile = "DRAGON";
+            attackProfiles[] = {"DRAGON"};
+        };
+    };
+    
+    class ShellBase;
+    class ace_m47_dragon_serviceCharge : ShellBase {
```

Comment:
looks like `GVAR(serviceCharge)` is better here and below, there are some `ace_m47_dragon_` strings

Reply:
Where else are the strings? I couldn't find any

Reply:
you've fixed all of them :)